js/_codemirror_unprintable.js

```diff
@@ -0,0 +1,67 @@
+// CodeMirror unprintable character extensions
+import { keymap } from "@codemirror/view";
+import { Decoration, ViewPlugin, WidgetType, MatchDecorator } from "@codemirror/view";
+import { EditorState, EditorView } from "./_codemirror";
```

Comment:
You may as well get these from the original export rather than making a circular import. At which point you have 6 from view and 1 from state

Reply:
Oh whoops, didn't notice that.